tests/test_annotation_stores.py

```diff
@@ -420,20 +706,28 @@ def test_sqlite_drop_index():
     assert "foo" not in store.indexes()
 
 
-def test_sqlite_drop_index_fail():
+def test_sqlite_drop_index_fail() -> None:
     """Test dropping an index that does not exist."""
     store = SQLiteStore()
     with pytest.raises(sqlite3.OperationalError):
         store.drop_index("foo")
 
 
-def test_sqlite_optimize_no_vacuum(fill_store, tmp_path):
+def test_sqlite_optimize_no_vacuum(fill_store, tmp_path) -> None:
     """Test running the optimize function on an SQLiteStore without VACUUM."""
     _, store = fill_store(SQLiteStore, tmp_path / "polygon.db")
     store.optimize(limit=0, vacuum=False)
 
 
-def test_annotation_to_geojson():
+def test_sqlite_wkb(fill_store) -> None:
```

Comment:
Please can you specify the annotation type for fill_store?

Reply:
Again as with monkeypatch it has an even more complex type that yields a callable at runtime.

Reply:
This could possibly be hinted sensibly using protocols. But this is beyond the scope of this PR.

Reply:
ok, can we add this in typing?

Reply:
Yes possibly although it would be specific to tests and not used in the main codebase so maybe we should put it in conftest or a tests/typing.py file?